src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAloneOrSingleline.java

```diff
@@ -138,14 +138,14 @@ void doDoubleBraceInitialization() {
             add("AB21/X");
             add("YYLEX");
             add("AR5E");
-        }});  //it's ok, can't be formatted better
+        }});  //violation
```

Comment:
did you mean "No violation" ? this is the  same case as above ( in diff ).
Looks like I forgot some nuance or ..... please explain.

Reply:
It is a violation and looks correct to me.

````
 "141:9: " + getCheckMessage(MSG_KEY_LINE_ALONE, "}", 9),
````

Reply:
@rnveach , what is a difference with line 108 - https://github.com/checkstyle/checkstyle/pull/4247/files/3510f1ee0b09371d3f7b6154e34e2859ebc3c5d7#diff-a7e700698e09368ce7408af88bf91357R108

Reply:
The difference is at line 108 the 2nd `}` is alone on the line. Line 141 has a `)` after it and since the setting is alone or singleline, the `)` should be on a different line.